app/helpers/edit_content_helper.rb

```diff
@@ -0,0 +1,24 @@
+module EditContentHelper
```

Comment:
I feel this module name is a bit ambiguous/confusing, maybe something like `ExternalLinksHelper`?

Reply:
I'm ok with that, and I've changed the naming.